ledger/ledger-api-bench-tool/src/test/suite/scala/com/daml/ledger/api/benchtool/submission/CommandSubmitterITSpec.scala

```diff
@@ -0,0 +1,307 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
```

Comment:
I think this is a decent effort to integration test this code! Although not sure we need this level of sophistication for this tool.

Reply:
Extracted it to separate PR: https://github.com/digital-asset/daml/pull/13533/files